application/.gitignore

```
@@ -12,4 +12,5 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 supervisord.*
-.php_cs.cache
\ No newline at end of file
+.php_cs.cache
+docker-compose.yml
```

```suggestion
.php_cs.cache
```

docker-compose.yml ファイルはgitで管理するので、.gitignoreに含めません。